### PR TITLE
Kubevirt: vm template details: Fix base template link and label

### PR DESCRIPTION
**Pull request by yaacov**
In `VMTemplateLink` the `template` prop may not be a `TemplateKind` , if called for `base template`  it may be an object with fields `name` and `namespace`.

Before:
OKD(1)

After:
OKD

### CODE REVIEW

**Comment by yaacov** [comment]
@mareklibra @vojtechszocs @suomiy @rawagner please review.

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx [hunk 1]
```diff
@@ -7,8 +7,8 @@ import { TemplateModel } from '@console/internal/models';
 import { TemplateKind } from '@console/internal/module/k8s';
 
 export const VMTemplateLink: React.FC<VMTemplateLinkProps> = ({ template }) => {
-  const name = getName(template);
-  const namespace = getNamespace(template);
+  const name = template.name || getName(template);
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx]: can we check for null here please?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx]: :+1:

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx [hunk 2]
```diff
@@ -7,8 +7,8 @@ import { TemplateModel } from '@console/internal/models';
 import { TemplateKind } from '@console/internal/module/k8s';
 
 export const VMTemplateLink: React.FC<VMTemplateLinkProps> = ({ template }) => {
-  const name = getName(template);
-  const namespace = getNamespace(template);
+  const name = (template && template.name) || getName(template);
+  const namespace = (template && template.namespace) || getNamespace(template);
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx]:
```suggestion
  const namespace = template && (template.namespace || getNamespace(template));
```

makes more sense IMO

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx]: Ahh ... yes ... sorry ...

Thanks :exclamation: fixing :-)

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx]: done

**Pull request merged**